api-reference/commands/runflow.md

```diff
@@ -57,7 +57,7 @@ appId: com.example.app
 - runFlow: 
     file: anotherFlow.yaml
     env:
-      MY_PARAMETER="123"
+      MY_PARAMETER: "123"
```

Comment:
Would the integer example be more clear without the quotes? I'm comparing to the `PASSWORD` example for [inline parameters](https://maestro.mobile.dev/advanced/parameters-and-constants#inline-parameters).

```suggestion
      MY_PARAMETER: 123
```

Reply:
@matthewpalsich I kept the existing syntax the same and just fixed the colon. I believe these env variables are always passed as Strings, so probably safer to keep the quotes, but happy to let the team lead on this one.